doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -46,7 +58,7 @@ The default parameters for ``temperature`` and ``convection-radiation`` are show
 * ``type``: type of boundary condition being imposed. At the moment, choices are:
     * ``noflux`` (default) so that there is no heat transfer boundary condition,
     * ``temperature`` (Dirichlet BC), to impose a given temperature ``value`` at the boundary,
-    * ``convection-radiation`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` and emissivity of the boundary :math:`\mathbf{\epsilon}` following Newton's law of cooling (and heating) and Stefan-Boltzmann law of radiation.
+    * ``convection-radiation`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` and emissivity of the boundary :math:`\mathbf{\epsilon}` following Newton's law of cooling (and heating) and Stefan-Boltzmann law of radiation. Note that the expressions for ``h``, ``Tinf`` and ``emissivity`` can be time dependent, but the current implementation doesn't allow for space dependence (the expressions are evaluated at the origin).
```

Comment:
This could be fixed in the future quite easily I believe

Reply:
Would it be ok to do the functions evaluation at the face barycenter, and add the value to the HT scratch data for the assembly? I can do it now if you'd like

Reply:
Well in this case I would calculate it at the face gauss point location and add it to the scratch data. it would be that simple really. Just like we do with the Nitsche weak immersed boundary in NS.